keithleyClient/keithleyInterface.py

```diff
@@ -416,6 +418,9 @@ def isTriped(self,statusword):
         bit = 0x08
         if int(statusword)&bit == bit:
 	    print 'keithley is tripped'
+            self.clearErrorQueue()
+            self.clearBuffer()
+            time.sleep(1)
```

Comment:
Our power supplies were having trouble recovering after tripping, but this cures it